changelog/unreleased/bug-fixes/2906--proper-display-of-duration-types.md

```diff
@@ -0,0 +1,6 @@
+Duration types with trailing '9' digits are now displayed correctly. The
+duration of 1.999 seconds was displayed as "1.1" and now it is shown as "2.0".
+VAST at most displays two digits after decimal point for duration types and
+this causes the values to be rounded respectively before being displayed.
+Previously VAST rounded down digits from 0 - 4 and now it does that from the
+0 - 5 range.
```

Comment:
Here's a suggestion for a rephrasing.

> We fixed incorrect printing of human-readable durations in some edge cases. E.g., the value 1.999s was rendered as 1.1s instead of the expected 2.0s. This bug affected the JSON and CSV export formats, and all durations printed in log messages or the status command.

I wouldn't describe how VAST renders and rounds the data. For the user-facing changelog what's relevant is what this affected and that it is now fixed.